dist/tools/cpy2remed/cpy2remed.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+#cpy2remed - copy to removable media
+#$1 contains generated hexfile
+#$2 contains directory name for this particular nucleo board which could be
+#   set in the board makefile.include using variable DIR_NAME_AT_REMED
+
+HEXFILE="$1"
+DEV_DIR="$2"
+
+REMED_MOUNT_PATH="${REMED_MOUNT_PATH:-/media/$USER/}$DEV_DIR/"
```

Comment:
This looks an awful lot like what `uf2conv` is doing 

Does `PROGRAMMER=uf2conv` work for this board?

Reply:
I checked this a minute ago and UF2 file produced by uf2conv.py script is not recognize by the ST-LINK on-board programmer. The ST-LINK needs intel HEX. Yes, this programmer is similar to uf2conv and to elf2uf2 RIOT programmers. All works similar, but in all three cases there are other file formats. What is interesting (complicated) UF2 file format needed by Raspberry Pi Pico is different that UF2 format produced by uf2conv tools (see PR #17348 ).